components_library/processors/simple_core.py

```diff
@@ -95,3 +97,7 @@ def connect_interrupt(
             self.core.interrupts[0].pio = interrupt_requestor
             self.core.interrupts[0].int_requestor = interrupt_responce
             self.core.interrupts[0].int_responder = interrupt_requestor
```

Comment:
Does gem5 work ok if you set these to None?

Reply:
Hmm, probably not. They are required when x86 and not required otherwise. Maybe instead the condition should be `if interrupt_requestor or interrupt_responder` or something like that instead of checking for x86 multiple places.

Reply:
Don't forget to implement this :)

Reply:
Do I need to do something more right now? Currently, it's checking to see if it's x86. We could probably clean that up, but it's not currently in my scope. I have a few other open things in this PR that I'll try to come back to once I've pushed this relation chain upstream.

Reply:
Ok. It's probably fine right now.